exporter/lokiexporter/README.md

```diff
@@ -0,0 +1,62 @@
+# Loki Exporter
+
+Exports data via HTTP to [Loki](https://grafana.com/docs/loki/latest/). By default, this exporter requires TLS and
```

Comment:
Can an exporter require TLS? I guess it's servers that require it, not exporters. Here it will be TLS or not based on whether endpoint is https or http I think. Or if plaintext really isn't supported, then can you add to the docs for `endpoint` that the URL must start with `https://`?

Reply:
After a bit more thought, I have removed the requirements for https by default, since some Loki instances my be colocated in the same network as the Otel collector and organizations may not require TLS within the same network.

Updating docs now.